open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/CommonHandler.java

```diff
@@ -5,24 +5,23 @@
 import org.odpi.openmetadata.accessservices.assetlineage.AssetContext;
 import org.odpi.openmetadata.accessservices.assetlineage.GraphContext;
 import org.odpi.openmetadata.accessservices.assetlineage.LineageEntity;
+import org.odpi.openmetadata.accessservices.assetlineage.ffdc.AssetLineageErrorCode;
 import org.odpi.openmetadata.accessservices.assetlineage.util.Converter;
 import org.odpi.openmetadata.commonservices.ffdc.InvalidParameterHandler;
 import org.odpi.openmetadata.commonservices.repositoryhandler.RepositoryHandler;
 import org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException;
 import org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException;
 import org.odpi.openmetadata.frameworks.connectors.ffdc.UserNotAuthorizedException;
-import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.EntityDetail;
-import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.Relationship;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.*;
 import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.typedefs.TypeDef;
 import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.repositoryconnector.OMRSRepositoryHelper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import sun.security.util.ManifestEntryVerifier;
```

Comment:
We should not use sun.* classes 

Java 11 is currently failing build due to this:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:jar (attach-javadocs) on project asset-lineage-server: MavenReportException: Error while generating Javadoc: 
[ERROR] Exit code: 1 - /home/vsts/work/1/s/open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/CommonHandler.java:20: error: package sun.security.util is not visible
[ERROR] import sun.security.util.ManifestEntryVerifier;
[ERROR]                    ^
[ERROR]   (package sun.security.util is declared in module java.base, which does not export it to the unnamed module)
[ERROR] 
[ERROR] Command line was: /usr/lib/jvm/zulu-11-azure-amd64/bin/javadoc -J-Xmx1536m @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/vsts/work/1/s/open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/target/apidocs' dir.
[ERROR] -> [Help 1]

```

Reply:
Thanks Nigel. Just removed it. :)